tikv/kv.go

```diff
@@ -571,6 +556,18 @@ func (s *KVStore) updateSafeTS(ctx context.Context) {
 			metrics.TiKVMinSafeTSGapSeconds.WithLabelValues(storeIDStr).Set(time.Since(safeTSTime).Seconds())
 		}(ctx, wg, storeID, storeAddr)
 	}
+
+	txnScopeMap := make(map[string][]uint64)
+	for _, store := range stores {
```

Comment:
why not keep the original way

Reply:
sorry, what do you mean by `original` here?

Reply:
`if store.IsLabelsMatch([]*metapb.StoreLabel{`

Reply:
`IsLabelsMatch` requires a specific value for the label, where here tries to precompute the min safe ts for all the labelled value